### ISSUE TITLE
Using Unity Sentis for Segmentation Model Only for Human

### ISSUE THREAD

**Issue by cancaglar** [ultralytics/ultralytics#14081]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello,
I have integrated a segmentation model using Sentis  in Unity and created a simple mask. However, I am curious about something: Is it possible to use this model exclusively for humans? I think this is called classification. (I am still new to these topics and learning.) And to be honest, I have researched what I want, but I probably haven't found a result because I don't know the correct terms.😅
Thank You!
Edit : My example code https://codeshare.io/r46vdY

![image](https://github.com/ultralytics/ultralytics/assets/59360242/3ffa3d15-bb7e-4bb2-b41f-ea0e722a4040)
![a](https://github.com/ultralytics/ultralytics/assets/59360242/19cdf02b-67b2-4d5c-87c8-545c0969239e)

**Comment by glenn-jocher** [comment]
@cancaglar hello,

Thank you for reaching out and for providing the example code! It's great to see your enthusiasm for learning and integrating segmentation models in Unity. 😊

To address your question, yes, it is indeed possible to use a segmentation model exclusively for detecting humans. This process involves training or fine-tuning your segmentation model to recognize only the human class. Here’s a concise guide on how you can achieve this:

1. **Dataset Preparation**: Ensure that your dataset contains images with human annotations. If you are using a pre-existing dataset like COCO, you can filter it to include only the human class.

2. **Model Training**: Train your segmentation model on this filtered dataset. If you are using YOLOv8, you can specify the dataset configuration to include only the human class. Here’s a quick example in Python:
    ```python
    from ultralytics import YOLO

    # Load a pretrained segmentation model
    model = YOLO("yolov8n-seg.pt")

    # Train the model on a dataset containing only human annotations
    model.train(data="path/to/human-only-dataset.yaml", epochs=100, imgsz=640)
    ```

3. **Inference**: When running inference, the model will now only detect and segment humans. You can then integrate this model into Unity using Sentis as you have done.

4. **Verification**: Ensure that the model is correctly integrated and verify its performance by testing it with various images to see if it accurately segments humans.

For more detailed steps on training and validating your segmentation model, you can refer to the [Ultralytics documentation on segmentation](https://docs.ultralytics.com/tasks/segment/).

If you encounter any issues or need further assistance, please ensure you are using the latest versions of the packages and provide a reproducible example as outlined [here](https://docs.ultralytics.com/help/minimum_reproducible_example).

Best of luck with your project, and feel free to reach out if you have any more questions!
